hc/front/tests/test_add_channel.py

```diff
@@ -37,5 +36,30 @@ def test_instructions_work(self):
             r = self.client.get(url)
             self.assertContains(r, "Integration Settings", status_code=200)
 
-    ### Test that the team access works
-    ### Test that bad kinds don't work
+
+class TeamAccessTestCase(BaseTestCase):
+    # Test that the team access works
+    def test_team_access_works(self):
+        kinds = ("email", "webhook", "pd", "pushover", "hipchat", "victorops")
+        for kind_name in kinds:
+            self.channel = Channel(user=self.alice, kind=kind_name)
+            self.channel.value = "test that team access works"
+            self.channel.save()
+
+            url = "/integrations/{}/checks/".format(self.channel.code)
+
+            self.client.login(username="bob@example.org", password="password")
```

Comment:
After looking at it, I think logging in once is enough and we don't need line 51 in the loop, we will move it out of the loop.